### PR TITLE
Doc multiple java agent extension  (#6132)

### PR DESCRIPTION
**Pull request by Roiocam**
motivation #6132

### CODE REVIEW

File: examples/extension/README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ To add the extension to the instrumentation agent:
           -Dotel.javaagent.extensions=build/libs/opentelemetry-java-instrumentation-extension-demo-1.0-all.jar
           -jar myapp.jar
      ```
+for multiple extensions, using jar files directory path for `otel.javaagent.extensions` value.
```
Comment (trask) [inline examples/extension/README.md:23]:
```suggestion
Note: to load multiple extensions, you can specify a directory path for the `otel.javaagent.extensions` value.
```

**Review by trask — APPROVED** [review]
thx!

cc @iNikem

**Review by anuraaga — APPROVED** [review]

**Review by laurit — APPROVED** [review]
@trask should we make `otel.javaagent.extensions` accept a comma separated list of paths to jar files?

**Comment by trask** [comment]
> @trask should we make `otel.javaagent.extensions` accept a comma separated list of paths to jar files?

makes sense, especially since its name is already pluralized `otel.javaagent.extensions` 👍

**Comment by Roiocam** [comment]
> @trask should we make `otel.javaagent.extensions` accept a comma separated list of paths to jar files?

@laurit @trask  I would love to handle this enhancement, please assign to me.

**Comment by laurit** [comment]
@Roiocam I already started working on it

**Pull request merged**